src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -18,7 +18,7 @@ import scala.language.postfixOps
 import scala.util.Random
 
 class TezosDatabaseOperationsTest
-  extends WordSpec
+    extends WordSpec
```

Comment:
I think we have different settings here...

Reply:
I'm quite astonished by how this class was totally reformatted, I simply changed a single attribute in the `proposals` table...

Yet the formatting can't be different, the definition's in the git repo's `.scalafmt.conf` file.
Unless scalafmt is bugged itself (I heard that it might be not consistent between runs in some case, yet I tried to re-run it and it consistently gave me this format)